src/main/java/org/hawkular/apm/qe/JaegerQEBase.java

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright 2015-2017 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.apm.qe;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.Invocation;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.MediaType;
+
+import org.hawkular.apm.qe.model.QESpan;
+
+import org.jboss.resteasy.spi.NotImplementedYetException;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import com.uber.jaeger.metrics.Metrics;
+import com.uber.jaeger.metrics.NullStatsReporter;
+import com.uber.jaeger.metrics.StatsFactoryImpl;
+import com.uber.jaeger.reporters.RemoteReporter;
+import com.uber.jaeger.reporters.Reporter;
+import com.uber.jaeger.samplers.ProbabilisticSampler;
+import com.uber.jaeger.samplers.Sampler;
+import com.uber.jaeger.senders.Sender;
+import com.uber.jaeger.senders.UDPSender;
+
+import io.opentracing.Tracer;
+
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * @author Kevin Earls 14 April 2017
+ */
+@Slf4j
+public class JaegerQEBase {
+    public static Integer JAEGER_FLUSH_INTERVAL = 100;
+    public static Integer JAEGER_MAX_PACKET_SIZE = 0;
+    public static Integer JAEGER_MAX_QUEUE_SIZE = 50;
+    public static Double JAEGER_SAMPLING_RATE = 1.0;
+    public static Integer JAEGER_PORT = 0;
```

Comment:
@kevinearls When we set `0` for some parameters will be set on default values here >> https://github.com/uber/jaeger-client-java/blob/master/jaeger-core/src/main/java/com/uber/jaeger/senders/UDPSender.java#L50~L61

However, we can go with our own default. This will avoid confusion on long run/new member.